docs/explanation/application-modelling.md

```diff
@@ -7,26 +7,22 @@ Juju provides simplicity, stability and security. Models reduce the cognitive ga
 
 Technical details such as CPU core counts, disk write throughput, and IP addresses are secondary. They are accessible to administrators, but an application model places the applications at the front.
 
-The primary function of a model is to enable you to maintain an uncluttered view of your service. Operational simplicity improves communication and understanding. Models provide an abstract view of the infrastructure that's hosting your service. 
```

Comment:
These kind of small but widely repeated changes are going to be painful in the next 3.6->main merge forward. I suggest doing so as soon as this PR lands so everything is fresher.